config/crds/apis.kcp.io_apiexports.yaml

```diff
@@ -467,8 +467,16 @@ spec:
                   workspaces up-to-date.
                 items:
                   properties:
+                    group:
+                      description: Group is the API group of the resource.
+                      type: string
+                    name:
+                      description: Name is the name of the resource.
+                      type: string
                     schema:
-                      description: Schema is the name of the referenced APIResourceSchema.
+                      description: |-
+                        Schema is the name of the referenced APIResourceSchema. This must be of the format
+                        "<version>.<name>.<group>".
```

Comment:
nit: version is misleading. Name does not really mean anything. Would use `<prefix>.<resource>.<group>`.